tests/distributed/test_dist_graph_store.py

```diff
@@ -0,0 +1,101 @@
+import os
+os.environ['OMP_NUM_THREADS'] = '1'
```

Comment:
Will we fix OPM_THREAD to 1 in our real tasks?

Reply:
do we know what the problem is?

Reply:
It is a bug of OMP and only exists in gcc compiler.